R/runClusterSummaryMetrics.R

```diff
@@ -40,16 +44,24 @@ runClusterSummaryMetrics <- function(inSCE, useAssay="logcounts", featureNames,
     warning("Specified genes '", toString(falseGenes), "' not found in ", warning)
   }
   
+  tempSCE <- inSCE[featureNames, ]
+
+  
+  if(isTRUE(scale)){
+    runNormalization(inSCE=tempSCE, useAssay=useAssay,scale = TRUE, normalizationMethod = NULL, transformation = NULL,
```

Comment:
Hi @akshayarav, I believe the runNormalization function returns an SCE object, so you might want to assign it back to the input object so scaled result is stored back.